plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/RuntimeUtils.java

```diff
@@ -302,6 +302,19 @@ public static boolean isWindows() {
         return IS_WINDOWS;
     }
 
+
+    /**
+     * Checks if current application is shipped from Windows store
+     * @return true if shipped from Windows store, false if not.
+     */
+    public static boolean isWindowsStoreApplication() {
+        if (!IS_WINDOWS) {
+            return false;
+        }
+        final String property = System.getProperty("dbeaver-windows-store-app");
+        return property != null && property.equalsIgnoreCase("true");
```

Comment:
```suggestion
        return CommonUtils.getBoolean(System.getProperty("dbeaver-windows-store-app"));
```

Additionally, consider the following:
1. Please use dots instead of hyphens in place of where you would put spaces: `dbeaver.windows-store-app`.
2. Please make a constant for this property name.